src/main/java/org/example/EasyAI.java

```diff
@@ -0,0 +1,28 @@
+package org.example;
+
+import java.util.Random;
+
+import static tictactoe.Main.checkWinner;
+
+public class EasyAI extends Player{
+    @Override
```

Comment:
usually I try to use 1 extra line in the beginning and in the end of the classes.